src/main/java/com/wojciechkocik/Application.java

```diff
@@ -0,0 +1,55 @@
+package com.wojciechkocik;
+
+import com.wojciechkocik.usage.dto.CourseUsageCreate;
+import com.wojciechkocik.usage.service.CourseUsageService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Random;
+
+@Slf4j
+@SpringBootApplication
+public class Application {
+
+    private static int DATA_ENTITIES_QUANTITY = 0;
```

Comment:
Would be better to use @Profile or environment variable. You should not change system behavior in different environments (production, development etc.) via hardcoded constant.